tidy3d/components/spice/sources/ac.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import pydantic.v1 as pd
+
+from tidy3d.components.base import Tidy3dBaseModel
+from tidy3d.components.types import ArrayFloat1D
+from tidy3d.constants import HERTZ, VOLT
+from tidy3d.constants import inf as td_inf
+
+
+class SSACVoltageSource(Tidy3dBaseModel):
+    """
+    Small-signal AC voltage source.
+
+    This source represents a small-signal AC excitation defined by a list of bias voltages,
+    a set of small-signal analysis frequencies, and a small-signal amplitude (linear scale).
+
+    Notes
+    -----
+    - The bias `voltage` refer to the DC operating point above the simulation ground.
```

Comment:
It should be RST syntax here
```
 ``voltage``
```
As a rule, we don't use markdown notation in the docs. It's either RST (two "\`"-s) in docstrings, or single quotes `'` in warning/error messages that will be printed to stdout.